packages/schematics/src/collection/library/index.ts

```diff
@@ -201,6 +201,29 @@ function addChildren(options: NormalizedSchema): Rule {
   };
 }
 
+function updateNgPackage(options: NormalizedSchema): Rule {
+  if (!options.publishable) {
+    return noop();
+  }
+  const dest = `${offsetFromRoot(options.projectRoot)}dist/@${options.prefix}/${
```

Comment:
Can we output this to dist/libs instead to match the sourcecode structure more?

```ts
const dest = `${offsetFromRoot(options.projectRoot)}dist/libs/${
```

Reply:
Sorry I wasn't clear, The result should be
```ts
const dest = `${offsetFromRoot(options.projectRoot)}dist/libs/${options.projectDirectory}`;
```

Reply:
May I ask the importance of having the npmScope specifically in the destination path? I think the primary issue is that they all get output to the same place. Now that we're using the projectDirectory, they all get put into their own section. Why have another directory to traverse to find your actual distribution files?

Reply:
I like that it mirrors the directory on npm or a private repository. People can see the dist and immediately know where the scope is without having to open the package.json or whatever else. For us, we have a private artifactory repo that just lists the packages.

But if not including the scope in the dist is preferable for the goal of this project. I can change it. I'll just make a workspace schematic to put the scope back into the dest for our requirements 😄